### PR TITLE
changes for dataport

### PR DESCRIPTION
**Pull request by AronKa-SAG**
- extraLabels specific for apigw
- formatted curl script
- quotes around initMemory, needs to be a string
- variable licensekey
- added subjectAltNames for certificates in ES
- fixed elasticsearch.nodesets
- in values: nodeSets and extraConfigSource as list

### CODE REVIEW

File: apigateway/helm/templates/deployment.yaml [hunk 1]
```diff
@@ -118,9 +112,9 @@ spec:
             value: {{ .Values.apigw.sagIsConfigProperties }}
           {{- include "common.metering.envs" . | nindent 10 }}
           - name: apigw_wrapper_java_initmemory
-            value: {{ .Values.apigw.initMemory }}
+            value: "{{ .Values.apigw.initMemory }}"
```
Comment (MarcFriedhoff) [inline apigateway/helm/templates/deployment.yaml:115]: better use | quote ?

File: apigateway/helm/values.yaml [hunk 2]
```diff
@@ -355,8 +358,9 @@ apigw:
   #   allowEdit: false
   #   properties:
   #     location: apigw-config.yml
-  extraConfigSource: {}
-  
+  extraConfigSource: []
```
Comment (MarcFriedhoff) [inline apigateway/helm/values.yaml:361]: Not sure - does this work in the template. Name should be changed to plural extraConfigSources if array.

**Review by MarcFriedhoff — COMMENTED** [review]
@AronKa-SAG  Can you please check my comments?

File: apigateway/helm/templates/deployment.yaml (hunk 1 above)
Comment (AronKa-SAG) [reply apigateway/helm/templates/deployment.yaml]: | quote added

File: apigateway/helm/values.yaml (hunk 2 above)
Comment (AronKa-SAG) [reply apigateway/helm/values.yaml]: changed name to extraConfigSources

File: apigateway/helm/templates/deployment.yaml [hunk 3]
```diff
@@ -25,6 +25,9 @@ metadata:
     {{- with .Values.extraLabels -}}
     {{ toYaml . | nindent 4 }}
     {{- end }}
+    {{- with .Values.apigw.extraLabels -}}
```
Comment (MarcFriedhoff) [inline apigateway/helm/templates/deployment.yaml:28]: das sollte nur .Values.extraLabels sein, oder?

**Review by MarcFriedhoff — APPROVED** [review]

File: apigateway/helm/templates/deployment.yaml (hunk 3 above)
Comment (AronKa-SAG) [reply apigateway/helm/templates/deployment.yaml:28]: works as intended

**Pull request merged**